.github/workflows/acceptance_tests.yml

```diff
@@ -0,0 +1,23 @@
+name: UpCloud Terraform provider acceptance tests
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Why not on PR:s?

Reply:
the acceptance tests take almost three hours to run 😁

Reply:
should probably look into that at some point as well

Reply:
Yep. I've been wondering about that too.